archive/archive_api/src/tests/test_archive_api.py

```diff
@@ -8,18 +8,17 @@ class TestReportIngestStatus:
     Tests for the GET /ingests/<guid> endpoint.
     """
 
-    def test_lookup_item(self, client, dynamodb_resource, table_name, guid):
-        table = dynamodb_resource.Table(table_name)
-        table.put_item(Item={'id': guid})
-
-        resp = client.get(f'/storage/v1/ingests/{guid}')
+    def test_lookup_item(self, client):
+        lookup_id = 'F423966E-A5E5-4D91-B321-88B90D1B5154'
+        resp = client.get(f'/storage/v1/ingests/{lookup_id}')
         assert resp.status_code == 200
-        assert json.loads(resp.data) == {'id': guid}
+        assert json.loads(resp.data) == {'progress': lookup_id}
 
```

Comment:
I think the AP returns the full ingest model https://github.com/wellcometrust/platform/tree/master/docs/rfcs/002-archival_storage#ingest-1